windows/hid.c

```diff
@@ -1003,6 +1030,16 @@ HID_API_EXPORT hid_device * HID_API_CALL hid_open_path(const char *path)
 	return dev;
 }
 
+static int hid_internal_write_output_report(hid_device *dev, const unsigned char *data, size_t length)
+{
+	BOOL res;
+	res = HidD_SetOutputReport(dev->device_handle, (void *)data, (ULONG)length);
+	if (res)
+		return (int)length;
+	else
+		return -1;
```

Comment:
either here or at the place of usage (in case of error) we need to register an error string

Reply:
Yep, good point. Added!